layers/+email/notmuch/packages.el

```diff
@@ -125,4 +133,15 @@
 (defun notmuch/init-helm-notmuch ()
   (use-package helm-notmuch
     :defer t
-    :init (with-eval-after-load 'notmuch)))
+    :init (progn
```

Comment:
`:init` does not need a `prog` form

Reply:
`progn` should be used if it has more than one statements/lines - as per [Spacemacs' convention](https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#use-package). And it should start on a newline under `:init`.

Reply:
I was aware of the convention, though I don’t agree with it. For one, it makes people think that this is a `use-package` requirement when it’s not. Also it serves no real purpose of visual clarity when you don’t have both an `:init` and a `:config`. But you are completely right, this is the convention and they should follow it. 

Also while having `progn` On its own line isn’t actually convention either, I agree it’s a good idea.